src/Prima/Pyxis/Modal.elm

```diff
@@ -606,31 +663,40 @@ updateOverlayOptions headerOptionsMapper modalOptions =
 render : Config msg -> Html msg
 render (Config modalConfig) =
     H.renderIf modalConfig.visible <|
-        overlay (pickOverlayOptions modalConfig)
-            [ modal (pickModalOptions modalConfig) modalConfig ]
+        overlay (pickOverlayOptions modalConfig) modalConfig
```

Comment:
Stai passando modalConfig due volte

Reply:
Era già così e l'ho lasciato, in quanto serve a `pickOverlayOptions`

```
pickOverlayOptions : ModalConfig msg -> OverlayOptions msg
pickOverlayOptions =
    pickModalOptions >> .overlayOptions
```

Penso sia stato lasciato. così per snellirre